src/inputs/EditableLabel/EditableLabel.js

```diff
@@ -22,18 +29,15 @@ const useStyles = makeStyles((theme) => ({
     overflow: 'hidden',
     textOverflow: 'ellipsis',
   },
-  textField: {
+  OutlinedInput: {
```

Comment:
To be replaced by `outlinedInput` to stay consistent with other classnames :wink: